framework_crates/bones_framework/src/networking.rs

```diff
@@ -469,6 +469,11 @@ where
                                         for (player_idx, (input, status)) in
                                             network_inputs.into_iter().enumerate()
                                         {
+                                            debug!(
+                                                "Net player({player_idx}) local: {}, status: {status:?}, input: {:?}",
+                                                self.local_player_idx == player_idx,
+                                                input
+                                            );
```

Comment:
Do you think we should we log this at a `trace` level instead, since it's rather verbose? It seems like we should make sure it's compiled out of release builds for performance sake, but I can't remember if we're already doing that, or at which log levels.

Reply:
I think you are right it may be more suitable as a trace level log than debug. Will change that.

I'm not sure we have any compile time filtering by default, only runtime. But tracing crate has [feature flags](https://docs.rs/crate/tracing/0.1.40/features) to disable log macros at compile time based on log level, and can set this separately for dev and release builds.